music21/stream/base.py

```diff
@@ -7110,7 +7110,8 @@ def extendDuration(self, objClass, *, inPlace=False):
     def stripTies(
```

Comment:
this is getting too big -- I'm trying to reduce the size of stream/base.py (it's starting to choke certain checkers) -- so my suggestion is to finish this review until it's basically done, then we'll move it to stream/tools.py and just leave it as a stump:

```
def stripTies(
    self,
    *,
    inPlace: bool = False,
    matchByPtch: bool = True,
    preserveVoices: bool = True,
):
    '''docs'''
    return tools.stripTies(self, inPlace=inPlace, matchByPitch...etc.)
```

this is what we've already been doing with makeTies(), etc. in stream/makeNotation.py

Thanks!